src/main/resources/hudson/plugins/validating_string_parameter/ValidatingStringParameterDefinition/config.jelly

```diff
@@ -22,22 +22,23 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
 
+<?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
 	xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form"
 	xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
     <f:entry title="${%Name}" help="/help/parameter/name.html">
-		<f:textbox field="name" />
+		<f:textbox name="parameter.name" value="${instance.name}" />

```

Comment:
I was trying to keep the whole file aligned with the real modification on line 42, which uses "name" and "value" rather than "field". Other than that, no particular reason. I have copied the feature from line 36 in: https://github.com/jenkinsci/jenkins/blob/master/core/src/main/resources/hudson/model/StringParameterDefinition/config.jelly#L36